app/controllers/catalog_controller.rb

```diff
@@ -218,15 +218,15 @@ class CatalogController < ApplicationController
     # since we aren't specifying it otherwise.
 
     config.add_search_field('search') do |field|
-      field.label = 'Everything'
+      field.label = I18n.t('search_fields.keyword')
```

Comment:
We backported the I18n for search + sort fields into Blacklight 6.13 here: https://github.com/projectblacklight/blacklight/pull/1815

Reply:
So I believe that means the path forward on this is to upgrade to Blacklight 6.13.0 and then add keys under the Blacklight 18n namespace: